plexpy/datafactory.py

```diff
@@ -1236,12 +1261,12 @@ def get_user_stats(self, rating_key=None, grouping=None):
                         'FROM session_history ' \
                         'JOIN session_history_metadata ON session_history_metadata.id = session_history.id ' \
                         'JOIN users ON users.user_id = session_history.user_id ' \
-                        'WHERE (session_history.grandparent_rating_key = ? ' \
-                        'OR session_history.parent_rating_key = ? ' \
-                        'OR session_history.rating_key = ?) ' \
+                        'WHERE (session_history.grandparent_rating_key IN %s ' \
+                        'OR session_history.parent_rating_key IN %s ' \
+                        'OR session_history.rating_key IN %s) ' \
                         'GROUP BY users.user_id ' \
-                        'ORDER BY total_plays DESC, total_time DESC' % group_by
-                result = monitor_db.select(query, args=[rating_key, rating_key, rating_key])
+                        'ORDER BY total_plays DESC, total_time DESC' % (group_by, rating_keys, rating_keys, rating_keys)
+                result = monitor_db.select(query)
```

Comment:
Just in case to protect against SQL injection.
```suggestion
                        'ORDER BY total_plays DESC, total_time DESC' % (
                            group_by, rating_keys_arg, rating_keys_arg, rating_keys_arg
                        )
                result = monitor_db.select(query, args=[rating_keys, rating_keys, rating_keys])
```